### ISSUE TITLE
Tag latest version

### ISSUE THREAD

**Issue by lenada** [mwillbanks/ZfcTwitterBootstrap#17]
@mwillbanks would you mind tagging your latest Version 0.2.1 in git?
Would be great for referencing a reliable state (something other then dev-master) in composer. Thank you :+1: 
Also the composer instructions in the README seem outdated, as it seems to be on packagist by now. Or is there a special reason to not just 

``` json
"require": {
   "mwillbanks/zfc-twitter-bootstrap": "dev-master"
}
```

should I send a PR to update the README?

**Comment by mwillbanks** [comment]
Absolutely; I will tackle this tomorrow.

**Comment by mwillbanks** [comment]
0.3.0 was tagged recently which included the flash messenger; 0.3.1 will be tagged at the end of the week since there have been vast improvements to it.
